Project/src/MakeCall/CallCard.js

```diff
@@ -60,6 +68,11 @@ export default class CallCard extends React.Component {
             remoteVolumeIndicator: undefined,
             remoteVolumeLevel: undefined,
             mediaCollector: undefined,
+
+            showParticipantsCard: false,
+            isSpotlighted: false,
```

Comment:
should this have a starting state?
e.g.
```suggestion
            isSpotlighted: this.spotlightFeature.<something>;
```

Reply:
Same withy raise hand below

Reply:
The starting state for both raisehand and spotlight for the local participant is always false